.github/workflows/ci.yml

```diff
@@ -70,7 +72,7 @@ jobs:
           echo "Revdeps to build: ${REVDEPS_TO_BUILD}"
           echo "revdeps=[${REVDEPS_TO_BUILD}]" >> "${GITHUB_OUTPUT}"
       - name: Write matrix output
-        if: matrix.arch == 'x64' && matrix.ocaml_version == '5.1.1'
+        if: matrix.arch == 'x64' && matrix.ocaml_version == '5.3.0'
```

Comment:
We should remove arch x86 now. This can be done after this PR is merged.

Reply:
Right. I can try that in an other PR

Reply:
Actually, partly done here in the CI file (I didn't renamed the docker image)